go.mod

```diff
@@ -3,26 +3,49 @@ module github.com/app-sre/deployment-validation-operator
 go 1.13
```

Comment:
Consider
```suggestion
go 1.16
```
Not because it actually does anything, but as a hint to contributors, since you are actually relying on 1.16 for $reasons.

Reply:
Yep, this I meant to do and forgot.  Thanks for catching.